source/fab/entry.py

```diff
@@ -0,0 +1,105 @@
+##############################################################################
+# (c) Crown copyright Met Office. All rights reserved.
+# For further details please refer to the file COPYRIGHT
+# which you should have received as part of this distribution
+##############################################################################
+"""
+Entry points for tools in the Fab suite of build related tools.
+"""
+import argparse
+import logging
+from pathlib import Path
+import sys
+
+import fab.application
+
+
+def fab_cli() -> argparse.Namespace:
+    """
+    Parses command line arguments for the core "fab" tool.
```

Comment:
The way we refer to the tool here as `the core "fab" tool` is inconsistent with the way we refer to it on L44 below, as `the core Fab build tool` - it doesn't matter much but could pick one and stick to it